src/test/java/com/rackspacecloud/metrics/queryservice/QueryServiceImplTests.java

```diff
@@ -212,7 +289,57 @@ private QueryResult getQueryResult() {
         return series;
     }
 
-    private TenantRoutes getTenantRoutes() {
-        return DevTestTenantRouteProvider.getTenantRoutes("1234");
+    private TenantRoutes getTenantRoutes(String tenantId, String measurement) {
+        ObjectMapper mapper = new ObjectMapper();
+        try {
+            TenantRoutes routes = mapper.readValue(ROUTES, TenantRoutes.class);
+            routes.setTenantIdAndMeasurement(String.format("%s:%s", tenantId, measurement));
+            return routes;
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        return null;
     }
+
+    private final static String ROUTES = "{\n" +
```

Comment:
If you want, here's my approach to read JSON into strings for unit tests

https://gist.github.com/itzg/50cd9b23cb4bc6f28ca4d45524a005a1